packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -202,6 +202,12 @@ message Trace {
 	// instead.
 	string signature = 19;
 
+	// Optional: when GraphQL parsing or validation against the GraphQL schema fails, these fields
+	// can include reference to the operation being sent for users to dig into the set of operations
+	// that are failing validation.
+	string operationBodyOnValidationFailure = 27;
```

Comment:
similarly, these shouldn't use "validation" to mean "parsing or validation". Split the fields or make a better name.

Maybe unexecutedOperationBody / unexecutedOperationName?

Reply:
I like it!